src/core/device.c

```diff
@@ -729,6 +730,9 @@ static bool device_is_ready(sd_device *dev) {
 
         assert(dev);
 
+        if (device_is_renaming(dev) > 0)
+                return false;
+
```

Comment:
btw, i wonder if we should have device_get_property_boolean() or so, that works a bit like getenv_bool(), i.e. parses things and returns -ENXIO if not set. And then we should probably use this for SYSTEMD_READY as well as for ID_RENAMING and all other cases like this